src/main/java/com/danit/security/SecurityConstants.java

```diff
@@ -2,12 +2,13 @@
 
 class SecurityConstants {
 
-  private SecurityConstants() {
-  }
-
   static final String SECRET = "fs4SecretKey";
   static final long EXPIRATION_TIME = 864_000_000; // 10 days
   static final String TOKEN_PREFIX = "Bearer ";
   static final String HEADER_STRING = "Authorization";
   static final String SIGN_UP_URL = "/api/users/auth";
+
+  private SecurityConstants() {
```

Comment:
Perhaps use a singleton pattern here?

Reply:
your propositions?

Reply:
Nevermind that :) I think I'm a bit tired